kubernetes/services/geth.yml.j2

```diff
@@ -30,6 +30,20 @@ spec:
               "--rpcapi",     "eth, web3, net",
               "--ipcdisable"
           ]
+          livenessProbe:
+            httpGet:
+              path: http://geth.default.svc.cluster.local/
+              port: 8545
+              httpHeaders:
+                - name:  Content-Type
+                  value: application/json
+            initialDelaySeconds: 5
+            periodSeconds:       8
+          readinessProbe:
+            exec:
+              command: ["/usr/local/bin/syncing-blockchain.sh"]
+            initialDelaySeconds: 70
+            periodSeconds:       10
```

Comment:
Isn't ` initialDelaySeconds: 70` a little pessimistic? What if Geth is up before that time?

Reply:
This is average time of geth is up and start a synchronization. The problem is that we don't wait for geth running, but for synchronization start up.

Reply:
It would be best to include this information in a comment.